express-api/package.json

```diff
@@ -45,10 +45,12 @@
     "zod": "3.23.3"
   },
   "devDependencies": {
+    "@eslint/js": "9.9.1",
```

Comment:
Are we still using this dependency? I don't see it imported.
Same with `@types/eslint__js` and the parent `typescript-eslint`

Reply:
> Are we still using this dependency? I don't see it imported. Same with `@types/eslint__js` and the parent `typescript-eslint`

Good catch. We don't use these dependencies. I'll remove it.